buildSrc/build.gradle.kts

```diff
@@ -23,4 +24,9 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
     implementation("org.jetbrains.dokka:dokka-gradle-plugin:$dokkaVersion")
     implementation("de.undercouch.download:de.undercouch.download.gradle.plugin:5.6.0")
+
+    implementation(gradleApi())
+    implementation(gradleKotlinDsl())
+    implementation(kotlin("gradle-plugin"))
+    implementation("org.gradle.kotlin:gradle-kotlin-dsl-plugins:4.2.1")  // Add this line
 }
```

Comment:
Please also sort imports from more generic one to more specific. 
I.e. start with gradle and kotlin imports, then say dokka and download.